osu.Game/Rulesets/UI/StandardHealthDisplay.cs

```diff
@@ -64,6 +93,21 @@ public StandardHealthDisplay()
             };
         }
 
-        protected override void SetHealth(float value) => fill.ScaleTo(new Vector2(value, 1), 200, EasingTypes.OutQuint);
+        public void BindProcessor(ScoreProcessor processor)

```

Comment:
rather than making the health display aware of the `ScoreProcessor`, how about just making a `Flash` method which gets bound directly?